### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by ShigekiKarita**
As contributors are increasing, it is time to add [CONTRIBUTING.md](https://help.github.com/articles/setting-guidelines-for-repository-contributors/) to share our guidelines. Here, I prepare the first draft based on [pytorch's](https://github.com/pytorch/pytorch/blob/master/CONTRIBUTING.md).

Editing and discussion are welcome.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+If you are interested in contributing to ESPnet, your contributions will fall into two categories:
+
+1. You want to propose a new Feature and implement it
+   post about your intended feature, and we shall discuss the design and implementation.
+   Once we agree that the plan looks good, go ahead and implement it.
+        
+2. You want to implement a feature or bug-fix for an outstanding issue
+   Look at the outstanding issues here: https://github.com/espnet/espnet/issues
+   Pick an issue and comment on the task that you want to work on this feature
+   If you need more context on a particular issue, please ask and we shall provide.
```
Comment (sw005320) [inline CONTRIBUTING.md]: > please ask and we shall provide

please ask us and then we shall provide more information.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+## Developing locally with ESPnet
+
+TBD
+
+## Unit testing
+
+ESPnet's testing is located under `test/`. Run the entire test suite with
+
+``` console
+$ pip install -r tools/test_requirements.txt
```
Comment (sw005320) [inline CONTRIBUTING.md]: I think this pip install is not required always, and may have some comments about it.

**Comment by sw005320** [comment]
If you reflect my comments, you can merge this.

**Comment by ShigekiKarita** [comment]
Done

**Pull request merged**